ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/vlans.py

```diff
@@ -71,3 +87,10 @@ def _get_vlan_config(self, vlan, tenant) -> dict:
             vlans_vlan["trunk_groups"] = trunk_groups
 
         return vlans_vlan
+
+    def _raise_duplicate_vlan_error(self, vlan_id: int, context: str, tenant_name: str, duplicate_vlan_config: dict):
```

Comment:
The proper return type for raising function is NoReturn - https://docs.python.org/3/library/typing.html#typing.NoReturn

Reply:
Fixed on all of them